### ISSUE TITLE
learning rate can't be set?

### ISSUE THREAD

**Issue by alwaysPKU** [balancap/SSD-Tensorflow#280]
when i train without checkpoint , the learning rate is default == 0.01 , and can't be reset...
and the loss = nan

**Comment by StevenBanama** [comment]
reducing learning rate to 0.0001,  which is 0.01 as default , and then the total loss may continue decay.

**Comment by alwaysPKU** [comment]
> reducing learning rate to 0.0001, which is 0.01 as default , and then the total loss may continue decay.

excuse me， what't your total loss at last? is it nearly 0?

**Comment by StevenBanama** [comment]
> > reducing learning rate to 0.0001, which is 0.01 as default , and then the total loss may continue decay.
> 
> excuse me， what't your total loss at last? is it nearly 0?
The total loss reach 40 without fine tuning checkpoint . But it reduces to 20 by fine tuning( just a few  attempts).

**Comment by alwaysPKU** [comment]
> > > reducing learning rate to 0.0001, which is 0.01 as default , and then the total loss may continue decay.
> > 
> > 
> > excuse me， what't your total loss at last? is it nearly 0?
> > The total loss reach 40 without fine tuning checkpoint . But it reduces to 20 by fine tuning( just a few  attempts).

i don’t know why the loss is so big， isn't it should be a number close to zero?

**Comment by StevenBanama** [comment]
you can get the location loss, and then read SSD loss, where you may find the reason.

**Comment by rainley123** [comment]
> reducing learning rate to 0.0001, which is 0.01 as default , and then the total loss may continue decay.

I have a question, my loss is 20-40 , but the accuracy of my model is nearly 0, why this happen

**Comment by StevenBanama** [comment]
I have a question, my loss is 20-40 , but the accuracy of my model is nearly 0, why this happen
maybe you use bad paramters, such as bad learning rate, low iteration, bad anchors. We use dataset of pascal2007, 0.001 for 60k iterator , 0.0001 for 80k, e-5 for 40k,  mAP reach 54%(mobilenetv2) without pretaining and total loss is 18.  It may help you, but i cant reproduce the result of the paper without pretaining.
